### PR TITLE
[jit] fix indexing into a tensor with a tuple

### PR DESCRIPTION
**Pull request by suo**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #79335

As title.

Differential Revision: [D37089015](https://our.internmc.facebook.com/intern/diff/D37089015)

### CODE REVIEW

**Comment by suo** [comment]
@suo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D37089015).

**Comment by suo** [comment]
@suo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D37089015).

File: torch/csrc/jit/frontend/ir_emitter.cpp [hunk 1]
```diff
@@ -3639,8 +3639,12 @@ struct to_ir {
         }
         auto input =
             emitSugaredExpr(apply.inputs()[0], 1)->asValue(loc, method);
-
-        return std::make_shared<SimpleValue>(emitIndex(loc, self, {input}));
+        if (input->type()->kind() == TypeKind::TupleType) {
+          return std::make_shared<SimpleValue>(
+              emitIndex(loc, self, createTupleUnpack(input)));
+        } else {
```
Comment (gmagogsfm) [inline torch/csrc/jit/frontend/ir_emitter.cpp:3645]: Nit, early return makes `else` unnecessary.
if (xx):
  return yy

return zz

**Review by gmagogsfm — APPROVED** [review]

**Comment by suo** [comment]
@suo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D37089015).

**Pull request closed without merging**